src/mainwindow.cpp

```diff
@@ -593,13 +606,14 @@ void MainWindow::setupLine()
 void MainWindow::setupRightFrame()
 {
     ui->frameRightTop->installEventFilter(this);
+    setCSSThemeAndUpdate(ui->frameRight, m_currentTheme);
 
-    QString ss = QStringLiteral("QFrame{ "
-                                "  background-color: %1; "
-                                "  border: none;"
-                                "}")
-                         .arg(m_currentRightFrameColor.name());
-    ui->frameRight->setStyleSheet(ss);
+    // QString ss = QStringLiteral("QFrame{ "
```

Comment:
Something strange going on over here: when I comment this block of code, the application segfaults when resizing. Specifically, it segfaults when calling `setLineWrapColumnOrWidth` (line 911) in current revision, which doesn't make much sense to me. My setCSSThemeAndUpdate function (which is what replaces the commented code) seems to work fine and changes the background color and border.

I don't get how this segfault is related to setting the stylesheet manually like how it's being done in the commented code. Any ideas?

Reply:
Seems to only happen on Qt5 :(